### PR TITLE
[11.0][IMP]mrp_multi_level

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
* let third modules to choose whether to continue exploding or not

This is useful for subcontracting and potentially for other flows.

@ForgeFlow

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
@LoisRForgeFlow are you ok with this?

File: mrp_multi_level/models/product_mrp_area.py [hunk 1]
```diff
@@ -109,6 +109,7 @@ class ProductMRPArea(models.Model):
         string="Group Days of Estimates",
         default=1,
     )
+    to_be_exploded = fields.Boolean(compute="compute_to_be_exploded")
```
Comment (LoisRForgeFlow) [inline mrp_multi_level/models/product_mrp_area.py]: no default implementation for `compute_to_be_exploded`? also, it should be a private method, could you add the underscore at the begging (`_compute_to_be_exploded`)

Comment (AaronHForgeFlow) [reply mrp_multi_level/models/product_mrp_area.py]: Oops did not commit all the changes. Thanks for letting me know.

**Comment by LoisRForgeFlow** [comment]
No need for any extra change? maybe an use of this field in the MRP multi level wizard?

**Comment by AaronHForgeFlow** [comment]
Yes, sorry, I messed up with git :fearful:

**Comment by LoisRForgeFlow** [comment]
One more question :laughing: why is this a field and not just a method? is the field going to be used in any view or have any purpose?

**Comment by AaronHForgeFlow** [comment]
No, the field is not used, it is just a technical help. I think yes, it could be a method.

**Comment by LoisRForgeFlow** [comment]
Then I would prefer a method :pray:

**Review by LoisRForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Thanks!

/ocabot merge patch

**Pull request merged**